docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx

```diff
@@ -55,9 +55,9 @@ After the token is created:
 To manage tokens for another user, select the user from the **Manage tokens for** dropdown:
 
 <Image
-src="/images/dagster-cloud/user-token-management/manage-user-tokens-for.png"
-width={1059}
-height={208}
+  src="/images/dagster-cloud/user-token-management/manage-user-tokens-for.png"
```

Comment:
@erinkcochran87 Will probably need to update this image after  https://github.com/dagster-io/internal/pull/3759/ goes out. I can hold off on landing this until then.

Reply:
There are a few places in the docs that mention this tab by name - I'll update this PR to grab those guys

Reply:
Nevermind, you got it. I'll just take my coffee and see myself out. 🤦‍♀️